src/Handler/CommandNameExtractor/SuffixedClassNameExtractor.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace League\Tactician\Handler\CommandNameExtractor;
+
+/**
+ * Extract the name from the class with a suffix, defaults to 'Handler'.
+ */
+class SuffixedClassNameExtractor implements CommandHandlerNameExtractor
+{
+    /**
+     * @var string
+     */
+    private $suffix;
+
+    /**
+     * Determine the suffix to use.
+     *
+     * @param string $suffix
+     */
+    public function __construct($suffix = 'Handler')
+    {
+        $this->suffix = $suffix;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function extract($command)
+    {
+        if (is_string($command) === false) {
```

Comment:
@mvriel 

Should we also assert here that `$command` is the name of an existing class?